src/stores/MessageReactionUserStore.js

```diff
@@ -0,0 +1,41 @@
+const DataStore = require('./DataStore');
+let User;
+
+/**
+ * A data store to store User models who reacted to a MessageReaction.
+ * @extends {DataStore}
+ */
+class MessageReactionUserStore extends DataStore {
+  constructor(client, iterable, data) {
+    if (!User) User = require('../structures/User');
+    super(client, iterable, User);
+    this.data = data;
```

Comment:
This is a reaction, and not something like a data packet ect. Can this be renamed to this.reaction.